### PR TITLE
Updates oneDNN version for AArch64 builds.

### PR DESCRIPTION
**Pull request by nSircombe**
Moves oneDNN version for AArch64 build from v2.6 to https://github.com/oneapi-src/oneDNN/commit/70d1198de554e61081147c199d661df049233279

Includes fixes to matmul broadcase which resolves a number
of unit test failures - https://github.com/oneapi-src/oneDNN/pull/1380

Removes superfluous patches for oneDNN 2.6.

### CODE REVIEW

**Comment by nSircombe** [comment]
Hi @penpornk,
This PR moves the oneDNN build forward to include a number of fixes on AArch64. This means we can remove a number of skipped tests from the CI/CD pipeline (those changes are also included in that patch). 
If this change look OK, please could we consider it for a cherry-pick onto the 2.10 release branch?

**Review by penpornk — APPROVED** [review]
Thank you for the PR! Let's get this merged into master first. I will try to cherry-pick this into 2.10 if things seem stable.

**Comment by nSircombe** [comment]
That sounds good to me, thanks @penpornk.

**Comment by penpornk** [comment]
@nSircombe Could you please help resolve the conflict? Thank you!

**Comment by nSircombe** [comment]
Hi @penpornk, 
The conflict should be resolved now.

**Review by gbaned — APPROVED** [review]

File: tensorflow/tools/ci_build/rel/ubuntu/cpu_arm64_pip.sh [hunk 1]
```diff
@@ -79,6 +79,7 @@ export TF_TEST_TARGETS="${DEFAULT_BAZEL_TARGETS} \
     -//tensorflow/python/kernel_tests/array_ops:split_op_test_cpu \
     -//tensorflow/python/kernel_tests/array_ops:split_op_test_gpu \
     -//tensorflow/python/kernel_tests/control_flow:scan_ops_test_cpu \
+    -//tensorflow/python/kernel_tests/control_flow:scan_ops_test_gpu \
```
Comment (penpornk) [inline tensorflow/tools/ci_build/rel/ubuntu/cpu_arm64_pip.sh:82]: These two lines are added by the latest merge commit, but they are not in master. Shouldn't they be under the `$TF_TEST_TARGETS` line instead?

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the update!

**Review by penpornk — APPROVED** [review]
Thank you for the update!

**Pull request merged**